src/peer_list.cpp

```diff
@@ -98,15 +98,15 @@ namespace libtorrent {
 
 	void peer_list::clear()
 	{
-		for (auto const p : m_peers)
+		for (auto* p : m_peers)
```

Comment:
Why not `for (auto* const p : m_peers)` in these 2 cases?

Reply:
it didn't seem all that important to make `p` const, since it's scope is very limited

Reply:
Well, I don't really have any arguments against the change besides "moar `const` moar better", so yeah.